src/pages/_app.js

```diff
@@ -56,6 +63,16 @@ BonusCalcApp.getInitialProps = async ({ ctx, Component: pageComponent }) => {
     return { accessDenied: true }
   }
 
+  configureScope((scope) => {
+    scope.addEventProcessor((event) => {
+      if (event.request.cookies[GSSO_TOKEN_NAME]) {
+        event.request.cookies[GSSO_TOKEN_NAME] = '[REMOVED]'
+      }
+
+      return event
+    })
+  })
```

Comment:
The documentation talks about using `beforeSend` in the global config - only problem is that the callback is only triggered for exceptions and not performance transactions. Adding an event processor works but if you use their `withSentry` helper it will add it's own that parses the request so you'd need to make sure that yours gets called afterwards. For other reasons I've not used the `withRouting` helper so thankfully we don't need to do that here.